api/oidc.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"fmt"
+)
+
+type OidcToken struct {
```

Comment:
I believe initialisms, such as "OIDC", should be in their natural-looking case. So this would be "OIDCToken". Similarly with the other types and functions. https://github.com/golang/go/wiki/CodeReviewComments#initialisms